### ISSUE TITLE
log.Fatal in the cli output

### ISSUE THREAD

**Issue by vieux** [docker/docker#13009]
Hello,

I see a `log.Fatal` in the cli with the current master

```
$ docker run scratch
Unable to find image 'scratch:latest' locally
FATA[0000] Error response from daemon: {"errorDetail":{"message":"'scratch' is a reserved name"},"error":"'scratch' is a reserved name"}
```

**Comment by vieux** [comment]
ping @dmcgowan @stevvooe

**Comment by hqhq** [comment]
Hi @vieux 
As the message said, `scratch` is a name reserved by Docker, you can see issue: https://github.com/docker/docker/issues/4242, and docs: http://docs.docker.com/articles/baseimages/#creating-a-simple-base-image-using-scratch

**Comment by vieux** [comment]
@hqhq I very well know what how to use scratch.

`Unable to find image 'scratch:latest' locally` is expected but

`FATA[0000] Error response from daemon: {"errorDetail":{"message":"'scratch' is a reserved name"},"error":"'scratch' is a reserved name"}` isn't (we shouldn't print JSON and a `log.Fatal` in the CLI

**Comment by vieux** [comment]
ping @jfrazelle @tiborvass

**Comment by duglin** [comment]
+1 - let's make the output more friendly

**Comment by runcom** [comment]
found where this could be possibly introduced

**Comment by jlhawn** [comment]
Related to https://github.com/docker/docker/pull/12777

**Comment by calavera** [comment]
Fixed by #12777
